csrankings-l.csv

```diff
@@ -367,6 +367,7 @@ Lexing Xie,Australian National University,https://cs.anu.edu.au/people/lexing-xi
 Leye Wang,Peking University,http://cs.pku.edu.cn/info/1174/2334.htm,o2I4sL8AAAAJ
 Leysia Palen,University of Colorado Boulder,http://cmci.colorado.edu/~palen,x0OsogEAAAAJ
 Li Chen 0009,Hong Kong Baptist University,http://www.comp.hkbu.edu.hk/~lichen,1kIvpVQAAAAJ
+Li Chen,University of Louisiana - Lafayette,https://lichenut.github.io/,-2tpbVgAAAAJ

```

Comment:
Needs disambiguation suffix (see https://dblp.org/pid/c/LiChen19.html).

Reply:
They are different.

Reply:
Li Chen 0019 is at Univ of Louisiana Lafayette.

Reply:
Added disambiguation suffix - "0019" for Li Chen